docs/capabilities.md

```diff
@@ -56,6 +56,8 @@ Capability | Description
 |`appium:showXcodeLog`|Whether to display the output of the Xcode command used to run the tests. If this is `true`, there will be **lots** of extra logging at startup. Defaults to `false`|e.g., `true`|
 |`appium:iosInstallPause`|Time in milliseconds to pause between installing the application and starting `WebDriverAgent` on the device. Used particularly for larger applications. Defaults to `0`|e.g., `8000`|
 |`appium:usePrebuiltWDA`|Skips the build phase of running the WDA app. Building is then the responsibility of the user. Only works for Xcode 8+. Defaults to `false`.|e.g., `true`|
+|`appium:usePreinstalledWDA`| Whether to launch a preinstalled WebDriverAgentRunner bundle id instead of running `xcodebuild` to start a WebDriverAgentRunner process. The default target bundle id is `com.facebook.WebDriverAgentRunner.xctrunner`. `appium:preInstalledWDABundleId` will customize the target. Defaults to `false`. |`true` or `true`|
```

Comment:
I assume we need a dedicated document that explains how to setup WDA, so it runs without xcodebuild

Reply:
and link to it here